forch/forchestrator.py

```
@@ -115,6 +122,8 @@ def _restore_states(self):
         self._faucet_events.set_event_horizon(event_horizon)
 
     def _restore_faucet_config(self, timestamp, config_hash):
+        if not config_hash:
```

Not a good place to do this because then it's just silently ignoring the error when somebody is calling restore_faucet_config with no config hashes. Rather, move this logic to the intermediate helper function and move the if check there.  Exact same logic, just encapsulated somewhere specific to event dispatching.